nistats/tests/test_glm_reporter.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+
+import warnings
+
+import nibabel as nib
+import numpy as np
+from nibabel import load
+from nibabel.tmpdirs import InTemporaryDirectory
+
+import pandas as pd
+
+from nistats._utils.testing import _write_fake_fmri_data
+from nistats.design_matrix import make_first_level_design_matrix
+from nistats.first_level_model import FirstLevelModel
+from nistats.reporting import glm_reporter as glmr
+from numpy.testing import dec
+
+from nistats.second_level_model import SecondLevelModel
+
+try:
+    import matplotlib as mpl
+except ImportError:
+    not_have_mpl = True
+else:
+    not_have_mpl = False
+
+
+@dec.skipif(not_have_mpl)
+def test_flm_reporting():
+    shapes, rk = ((7, 8, 7, 15), (7, 8, 7, 16)), 3
+    mask, fmri_data, design_matrices = _write_fake_fmri_data(shapes, rk)
+    flm = FirstLevelModel(mask_img=mask).fit(
+            fmri_data, design_matrices=design_matrices)
+    contrast = np.eye(3)[1]
+    report_flm = glmr.make_glm_report(flm, contrast, plot_type='glass',
+                                      height_control=None, min_distance=15,
+                                      alpha=0.001, threshold=2.78,
+                                      )
+    # catches & raises UnicodeEncodeError in HTMLDocument.get_iframe()
```

Comment:
what do you mean by this comment?

Reply:
`HTMLDocument.get_iframe()` is not able to handle certain unicode characters in Python2, like the greek alpha symbol and raises this error. Calling `HTMLDocument.get_iframe()` causes the tests to fail on Python2 if such a situation arises in future due to modifications.

I may add this comment here.

Reply:
I have noted this here as well:

https://github.com/nistats/nistats/blob/dec1dc53fb40dc0ccc4531e4a4d140ec0637aec1/nistats/reporting/glm_reporter.py#L793

Reply:
ok thanks

Reply:
it doesn't happen with python3?

Reply:
No python3 has much better unicode charset support